schemas/com.snowplowanalytics.mobile/remote_config/jsonschema/1-0-0

```diff
@@ -0,0 +1,161 @@
+{
+  "$schema" : "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "self" : {
+    "vendor" : "com.snowplowanalytics.mobile",
+    "name" : "remote_config",
+    "version" : "1-0-0",
+    "format" : "jsonschema"
+  },
+  "description": "The configuration file for the Snowplow mobile trackers.",
+  "type": "object",
+  "properties": {
+    "configurationVersion": {
```

Comment:
Do we need a configurationVersion? Doesn't the schema version give us that information already?

Reply:
Lets move this discussion back to https://github.com/snowplow/iglu-central/issues/1109 to unblock this release.